contracts/libs/MarginAccount.sol

```diff
@@ -10,12 +10,6 @@ import {SafeMath} from "../packages/oz/SafeMath.sol";
 library MarginAccount {
     using SafeMath for uint256;
 
-    // Account is a struct corresponding to a user that describes how many vaults that user has.
-    struct Account {
-        // the number of vaults a user has, starting at index 0 and incrementing. vaultCounter sorted chronologically.
-        uint256 vaultCounter;
-    }
-
```

Comment:
Don't remove this! I need it for invariant testing lol. I need to know how many vault counters are there on chain.

Reply:
We can just stored `mapping (address => uint256 )vaultCounter` in the controller.

Reply:
nvm its good! I saw the other new implementation